src/partials/how-its-made.html

```diff
@@ -0,0 +1,50 @@
+<main>
+  <div class="container">
+    <section>
+      <h2 class="tradition">tradition and love</h2>
+      <h1 class="how-is-made-title">how it’s made?</h1>
+
+      <div class="how-is-made-content">
+        <img
+          class="how-is-made-img"
+          src="./images/how-it-is-made1.png"
+          alt="the cow and milk"
+          width="510px"
```

Comment:
In html nu se trece px la dimensiuni, treci doar numarul, te rog sa stergi px de la width/height